.circleci/local_publish_helpers.sh

```diff
@@ -21,24 +21,33 @@ function uploadPkgCli {
     export version=$(./amplify-pkg-linux-x64 --version)
 
     if [[ "$CIRCLE_BRANCH" == "release" ]] || [[ "$CIRCLE_BRANCH" =~ ^run-e2e-with-rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^release_rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^tagged-release ]]; then
-        tar -czvf amplify-pkg-linux-arm64.tgz amplify-pkg-linux-arm64
+        if [[ ! "$CIRCLE_BRANCH" =~ ^tagged-release ]]; then
```

Comment:
changes in this file are from this PR: https://github.com/aws-amplify/amplify-cli/pull/11140 in order to be able to create the tagged release